xclim/core/dataflags.py

```diff
@@ -97,7 +108,7 @@ def _sanitize_attrs(da: xarray.DataArray) -> xarray.DataArray:
     return da
 
 
-@register_methods
+@register_methods()
```

Comment:
Why are parentheses needed here?

Reply:
I added a decorator level to the function: `@register_methods()` actually returns the real decorator, using the default value (None, thus `func.__name__`) as variable name.

Reply:
I don't get quite understand, but I'll give it a look on Monday with fresher eyes.